scanpy/plotting/anndata.py

```diff
@@ -1465,7 +1465,16 @@ def dotplot(adata, var_names, groupby=None, use_raw=None, log=False, num_categor
     size = (frac * 10) ** 2
 
     import matplotlib.colors
-    normalize = matplotlib.colors.Normalize(vmin=min(mean_flat), vmax=max(mean_flat))
+
+    #if user defined, then use the vmax, vmin keywords, else use data to generate them...
+    if ('vmax' in kwds) and ('vmin' in kwds):
```

Comment:
Frequently, I only set the vmax because I am fine with vmin (eg. 0). However, in the current code nothing will happen and this can be very confusing. 

Can you add a conditional for vmax and vmin separately?

Reply:
Hi Fidel, I pushed a new commit that takes vmin vmax as individual optional arguments and generates either from the data when they are not provided. Please see commit c0c28ee from my fork.